lms/static/js/bookmarks/views/bookmarks_list.js

```diff
@@ -60,7 +60,18 @@
             },
 
             visitBookmark: function (event) {
-                window.location = event.target.pathname;
+                var bookmark_id = $(event.currentTarget).data('bookmarkId');
+                var component_usage_id = $(event.currentTarget).data('usageId');
+                Logger.log(
+                    'edx.course.bookmark.accessed',
+                    {
+                       bookmark_id: bookmark_id,
+                       component_type: 'vertical',
```

Comment:
We are adding a field to the bookmark resource which will contain the type of the bookmark's block. So this will not be hardcoded.